### PR TITLE
Removed template from solution for CSES Planets Queries II

### PR DESCRIPTION
**Pull request by izhang05**

### CODE REVIEW

**Review by dongliu0426 — CHANGES_REQUESTED** [review]
use tabs

**Comment by izhang05** [comment]
ok I changed to tabs

**Review by dongliu0426 — APPROVED** [review]

File: solutions/CSES Planets Queries II.mdx [hunk 1]
```diff
@@ -260,74 +59,109 @@ bool incyc[MX];
...
+	}
 }
 
 void process() {
-    FOR(i,1,n+1) dfs(i);
-    FOR(i,1,n+1) par[i][0] = t[i];
-    FOR(i,1,18) FOR(j,1,n+1)
-        par[j][i] = par[par[j][i-1]][i-1];
-    FOR(i,1,n+1) if (incyc[i]) DFS(i);
+	for (int i = 1; i < n + 1; ++i) {
```
Comment (bqi343) [inline solutions/CSES Planets Queries II.mdx:85]: replace `< n+1` with `<= n` everywhere

File: solutions/CSES Planets Queries II.mdx [hunk 2]
```diff
@@ -260,74 +59,109 @@ bool incyc[MX];
...
+		if (incyc[i]) {
+			DFS(i);
+		}
+	}
 }
 
 int getPar(int a, int b) {
-    R0F(i,18) if (b&(1<<i)) a = par[a][i];
-    return a;
+	for (int i = (18) - 1; i >= 0; --i) {
```
Comment (bqi343) [inline solutions/CSES Planets Queries II.mdx:104]: why parentheses around 18?

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Comment by frodakcin** [comment]
I feel like those are more of personal preference? I think they don't cause any confusion

**Comment by bqi343** [comment]
> I feel like those are more of personal preference? I think they don't cause any confusion

Well, I'm still waiting for these changes ... why would you prefer `< n+1` over `<= n` ^_^

**Comment by frodakcin** [comment]
Well I'd prefer `<=n` but don't think it's significant enough to warrant requesting author to change?

**Comment by frodakcin** [comment]
Ohhh I see: A direct demacrofier was used so the weird code style is to be consistent with the macro as opposed to being author's choice.

Yea, in that case, it does makes sense to request getting rid of the extra stuff.

**Comment by bqi343** [comment]
Closing due to inactivity.

**Pull request closed without merging**